### PR TITLE
Upgrade dependencies and target frameworks

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,13 +2,10 @@ language: csharp
 
 os:
```
Comment (shargon) [inline .travis.yml]: We can move to github actions here too

Comment (erikzhang) [reply .travis.yml]: Yes. After this PR merged.

File: RpcNep5Tracker/RpcNep5Tracker.cs [hunk 2]
```diff
@@ -222,12 +219,12 @@ public bool ShouldThrowExceptionFromCommit(Exception ex)
                 if (++resultCount > _maxResults) break;
                 JObject transfer = new JObject();
                 transfer["timestamp"] = transferPair.Key.TimestampMS;
-                transfer["asset_hash"] = transferPair.Key.AssetScriptHash.ToArray().Reverse().ToHexString();
+                transfer["asset_hash"] = transferPair.Key.AssetScriptHash.ToArray().Reverse().ToArray().ToHexString();
```
Comment (erikzhang) [inline RpcNep5Tracker/RpcNep5Tracker.cs]:
```suggestion
                transfer["asset_hash"] = transferPair.Key.AssetScriptHash.ToString();
```

File: RpcNep5Tracker/RpcNep5Tracker.cs [hunk 3]
```diff
@@ -222,12 +219,12 @@ public bool ShouldThrowExceptionFromCommit(Exception ex)
...
                 transfer["transfer_address"] = transferPair.Value.UserScriptHash.ToAddress();
                 transfer["amount"] = transferPair.Value.Amount.ToString();
                 transfer["block_index"] = transferPair.Value.BlockIndex;
                 transfer["transfer_notify_index"] = transferPair.Key.BlockXferNotificationIndex;
-                transfer["tx_hash"] = transferPair.Value.TxHash.ToArray().Reverse().ToHexString();
+                transfer["tx_hash"] = transferPair.Value.TxHash.ToArray().Reverse().ToArray().ToHexString();
```
Comment (erikzhang) [inline RpcNep5Tracker/RpcNep5Tracker.cs]:
```suggestion
                transfer["tx_hash"] = transferPair.Value.TxHash.ToString();
```

File: RpcNep5Tracker/RpcNep5Tracker.cs [hunk 4]
```diff
@@ -268,12 +265,12 @@ private JObject GetNep5Balances(JArray _params)
...
             var dbCache = new DbCache<Nep5BalanceKey, Nep5Balance>(_db, null, null, Nep5BalancePrefix);
             byte[] prefix = userScriptHash.ToArray();
-            foreach (var storageKeyValuePair in dbCache.Find(prefix))
+            foreach (var (key, value) in dbCache.Find(prefix))
             {
                 JObject balance = new JObject();
-                balance["asset_hash"] = storageKeyValuePair.Key.AssetScriptHash.ToArray().Reverse().ToHexString();
-                balance["amount"] = storageKeyValuePair.Value.Balance.ToString();
-                balance["last_updated_block"] = storageKeyValuePair.Value.LastUpdatedBlock;
+                balance["asset_hash"] = key.AssetScriptHash.ToArray().Reverse().ToArray().ToHexString();
```
Comment (erikzhang) [inline RpcNep5Tracker/RpcNep5Tracker.cs]:
```suggestion
                balance["asset_hash"] = key.AssetScriptHash.ToString();
```

File: StatesDumper/StatesDumper.cs [hunk 5]
```diff
@@ -39,14 +39,12 @@ protected override bool OnMessage(object message)
         {
             if (!(message is string[] args)) return false;
             if (args.Length == 0) return false;
-            switch (args[0].ToLower())
+            return (args[0].ToLower()) switch
```
[the remark this answers is not in the document]
Comment (erikzhang) [reply StatesDumper/StatesDumper.cs]: It's "switch expression".

**Pull request merged**